### PR TITLE
enhance(schedules): add utc warning 

### PR DESCRIPTION
**Pull request by plyr4**
- enhance: add utc time warnings to schedules forms
- enhance: add utc time.now view to schedule forms
- refactor: move forms code to `Schedules/Form.elm` and fill out missing comments

Screenshot 2023-07-12 at 1 34 30 PM

### CODE REVIEW

**Review by timhuynh94 — APPROVED** [review]

**Review by timhuynh94 — DISMISSED** [review]

**Review by KellyMerrick — DISMISSED** [review]
lgtm

File: src/elm/Pages/Schedules/Form.elm [hunk 1]
```diff
@@ -41,16 +38,43 @@ import Html.Attributes
...
+-}
+viewEditForm : PartialModel a msg -> Html Msg
+viewEditForm model =
+    let
+        sm =
+            model.schedulesModel
+    in
+    div [ class "schedule-form", class "edit-form" ]
+        [ viewNameInput sm.form.name True
+        , viewValueInput sm.form.entry "0 0 * * * (runs at 00:00 AM in UTC)" (Util.toUtcString model.time)
```
Comment (ecrupper) [inline src/elm/Pages/Schedules/Form.elm:73]:
```suggestion
        , viewValueInput sm.form.entry "0 0 * * * (runs at 00:00 in UTC)" (Util.toUtcString model.time)
```
OR 
```suggestion
        , viewValueInput sm.form.entry "0 0 * * * (runs at 12:00 AM in UTC)" (Util.toUtcString model.time)
```

**Review by ecrupper — CHANGES_REQUESTED** [review]

File: src/elm/Pages/Schedules/Form.elm (hunk 1 above)
Comment (plyr4) [reply src/elm/Pages/Schedules/Form.elm:73]: good call! im going with 12:00 AM

**Review by ecrupper — DISMISSED** [review]

**Review by timhuynh94 — APPROVED** [review]

**Review by KellyMerrick — APPROVED** [review]
lgtm

**Pull request merged**